packages/eslint-plugin-pf-codemods/test/rules/v5/fileUpload-remove-onChange.js

```diff
@@ -0,0 +1,24 @@
+const ruleTester = require("../../ruletester");
+const rule = require("../../../lib/rules/v5/fileUpload-remove-onChange");
+
+ruleTester.run("fileUpload-remove-onChange", rule, {
+  valid: [
+    {
+      code: `import { FileUpload } from '@patternfly/react-core'; <FileUpload />`,
+    },
+    // No @patternfly/react-core import
+    { code: "<FileUpload />" },
```

Comment:
This should include the attribute, the reason for these tests is that we won't touch it if a user has a FileUpload that is not from PF.

Reply:
@kmcfaul  - I just noticed the same thing for your visiblity test, although the chances of another `ToolbarContent` with a wrongly spelled "visiblity" attribute is... pretty small

Reply:
Thanks for catching this!